README.md

```diff
@@ -1,38 +1,162 @@
 # Aksharamukha.js
-Aksharamukha in your browser!
 
-[![npm](https://img.shields.io/npm/v/aksharamukha)](https://www.npmjs.com/package/aksharamukha)
+Aksharamukha in your browser and Node.js!
 
 ## Description
-This project aspires to be a browser-compatible version of the [Aksharamukha](https://www.aksharamukha.com/) and its [Python library](https://github.com/virtualvinodh/aksharamukha-python), which is a transliteration tool for Indic scripts.
+This project aspires to be a browser-compatible version of the [Aksharamukha](https://www.aksharamukha.com/) and its [Python library](https://github.com/virtualvinodh/aksharamukha-python), which is a transliteration tool for Indic scripts. The goal is to enable users to perform script conversions directly in their web browsers or Node.js applications without needing to rely on server-side processing.
 
-The goal is to enable users to perform script conversions directly in their web browsers without needing to rely on server-side processing.
+#### Aksharamukha.js can be used in multiple ways: directly in the browser via CDN, as a Node.js library, or imported in frontend JS apps using modern bundlers.
```

Comment:
I think we can mostly skip these changes. The goal and intention don't change – we just want to specify other ways the package can be used.